docs/TektonHub.md

```diff
@@ -2,34 +2,64 @@
 
 TektonHub custom resource allows user to install and manage [Tekton Hub][hub].
 
-TektonHub is an optional component and currently cannot be installed through TektonConfig. It has to be installed seperately. It is available for both Kubernetes and OpenShift platform.
+TektonHub is an optional component and currently cannot be installed through TektonConfig. It has to be installed
+seperately. It is available for both Kubernetes and OpenShift platform.
 
 To install Tekton Hub on your cluster follow steps as given below:
 
-1.  Create the secrets for the API before we install Tekton Hub. By default,
-    Tekton Hub expects this secret to have the following properties:
+1. Fork the project [Tekton Hub][hub] to your Git handle to update
```

Comment:
it is not mandatory to fork hub repo, isn't it?
hub just need that file, it can be in any repo just need a url to acess it

Reply:
yes the file can be anywhere, 
but it would be easier for the user to just edit, commit and push the file instead of copying the file or creating the file 
but it is not necessary for the user to fork the project

Reply:
well end user don't have to fork hub repo in their org just to maintain configuration
we should mention the need is for configuration file which you can maintain in any repo accessible to hub api server
and forking could be a solution mentioned after that